src/components/app/MenuButtons/index.css

```diff
@@ -14,7 +14,6 @@
   padding: 0 20px;
   border-left: 1px solid var(--grey-30);
   color: var(--grey-90);
-  cursor: default;
```

Comment:
This came from a discussion with @fqueze

Reply:
I wonder if we should do this on all clickable buttons actually (eg: profile info, publish, etc), what do you think?

Reply:
What was the discussion? I'm not really that opinionated but I believe I was following the general practice of DevTools not using pointer cursors in general.

As for clickable buttons, I'd rather have the semantically correct element here.

Reply:
The issue was that Florian was surprised that clicking on Docs opens a new tab (despite the icon), and that the cursor could help.

> As for clickable buttons, I'd rather have the semantically correct element here.

I believe we do have the correct elements here. A link for the navigation and buttons for opening panels. The initial cursor for buttons is the "default", while for links this is the "pointer". I believe we should make it the pointer cursor for all buttons. For example this is what github does on the buttons for comments :)